### PR TITLE
Sejin EAT-1010M2 keyboard

### PR DESCRIPTION
**Pull request by DmNosachev**
Sejin Electronics EAT-1010M2 controller replacement.
Handwired Blue Pill to membrane.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/sejin_eat1010r2/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+/*
+void matrix_init_user(void) {
+}
+void matrix_scan_user(void) {
+}
+bool led_update_user(led_t led_state) {
+    return true;
+}
+*/
```
Comment (waffle87) [inline keyboards/handwired/sejin_eat1010r2/keymaps/default/keymap.c:58]:
```suggestion
```

File: keyboards/handwired/sejin_eat1010r2/readme.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# QMK-based firmware for Zenith Z-150 keyboard
+
+![Z-150](https://imgur.com/jayyyvJ.jpg)
```
Comment (waffle87) [inline keyboards/handwired/sejin_eat1010r2/readme.md:3]:
```suggestion
![Z-150](https://i.imgur.com/jayyyvJh.jpg)
```

File: keyboards/handwired/sejin_eat1010r2/readme.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+Zenith Z-150 keyboard conversion project: direct connection of Black Pill to the matrix.
+
+* Keyboard Maintainer: [DmNosachev](https://github.com/DmNosachev)
+* Hardware Supported: [Zenith Z-150 keyboard, 100-1860 version with white logo](https://deskthority.net/wiki/Zenith_Z-150), Black Pill STM32F103C8T6 MCU board. Alternatevely you can use any MCU which is supported by QMK and has 22 or more IO pins: Arduino Micro, Teensy 2.0, Teensy 2.0++, Blue Pill, etc.
+
+Earlier 100-1886 version with black logo has different PCB.
+
+Make example for this keyboard (after setting up your build environment):
+
+    make handwired/z150:default
```
Comment (waffle87) [inline keyboards/handwired/sejin_eat1010r2/readme.md:14]:
```suggestion
    make handwired/sejin_eat1010r2:default

Flashing example for this keyboard:

    make handwired/sejin_eat1010r2:default:flash

```

File: keyboards/handwired/sejin_eat1010r2/readme.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
```
Comment (waffle87) [inline keyboards/handwired/sejin_eat1010r2/readme.md:17]: Please add instruction on entering bootloader mode, as per [template](https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md?plain=1#L21-L27).

File: keyboards/handwired/sejin_eat1010r2/readme.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+LEDs (from left to right): NumLock, CapsLock, ScrollLock:
+![Z-150 LED cathodes](https://imgur.com/MO5NLlZ.jpg)
+
+Their anodes are connected to VCC through 220Ω resistors.
+![Z-150 LED anodes](https://imgur.com/w1MfjRN.jpg)
+
+You can keep the original resistors, but with Black Pill or any other 3.3V controller the LEDs may seem a little dim. Try 150Ω or close value if you want to fix that.
+
+
+## Black Pill
```
Comment (waffle87) [inline keyboards/handwired/sejin_eat1010r2/readme.md:41]: `info.json` says this uses a bluepill? May want to clear that up.

File: keyboards/handwired/sejin_eat1010r2/info.json [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+        "midi": false,
+        "mousekey": true,
+        "nkro": false,
+        "rgblight": false,
+        "unicode": false,
+        "bluetooth": false,
+        "backlight": false,
+        "sleep_led": false
+    },
+	"debounce": 5
```
Comment (waffle87) [inline keyboards/handwired/sejin_eat1010r2/info.json:32]:
```suggestion
```

File: keyboards/handwired/sejin_eat1010r2/info.json [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+}
```
Comment (waffle87) [inline keyboards/handwired/sejin_eat1010r2/info.json:33]: This file is missing the x, y coordinates (at the minimum). See [configuator support](https://docs.qmk.fm/#/reference_configurator_support).

File: keyboards/handwired/sejin_eat1010r2/config.h [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+/* key matrix size */
+#define MATRIX_ROWS 8
+#define MATRIX_COLS 16
+
+#define MATRIX_HAS_GHOST
+
+#define NUM_LOCK_LED_PIN B0
+#define SCROLL_LOCK_LED_PIN B10
+#define CAPS_LOCK_LED_PIN B1
+
```
Comment (waffle87) [inline keyboards/handwired/sejin_eat1010r2/config.h:31]:
```suggestion
#define MATRIX_HAS_GHOST
```

File: keyboards/handwired/sejin_eat1010r2/info.json [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+        "extrakey": true,
+        "midi": false,
+        "mousekey": true,
+        "nkro": false,
+        "rgblight": false,
+        "unicode": false,
+        "bluetooth": false,
+        "backlight": false,
+        "sleep_led": false
+    },
```
Comment (waffle87) [inline keyboards/handwired/sejin_eat1010r2/info.json:31]:
```suggestion
        "command": false,
        "console": false,
        "extrakey": true,
        "midi": false,
        "mousekey": true,
        "nkro": false,
        "rgblight": false,
        "unicode": false,
        "bluetooth": false,
        "backlight": false,
        "sleep_led": false
    },
    "indicators": {
        "num_lock": "B0",
        "caps_lock": "B1",
        "scroll_lock": "B10",
    },
```

File: keyboards/handwired/sejin_eat1010r2/sejin_eat1010r2.c [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+
+bool led_update_kb(led_t led_state) {
+    bool res = led_update_user(led_state);
+    if(res) {
+        writePin(NUM_LOCK_LED_PIN, !led_state.num_lock);
+        writePin(CAPS_LOCK_LED_PIN, !led_state.caps_lock);
+        writePin(SCROLL_LOCK_LED_PIN, !led_state.scroll_lock);
+    }
+    return res;
+}
```
Comment (waffle87) [inline keyboards/handwired/sejin_eat1010r2/sejin_eat1010r2.c:39]:
```suggestion
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

**Comment by DmNosachev** [comment]
Sorry, I messed up something in my fork. I'd better delete this pull request and fix everything.

**Pull request closed without merging**